types/node/querystring.d.ts

```diff
@@ -1,11 +1,11 @@
 declare module "querystring" {
     interface StringifyOptions {
-        encodeURIComponent?: Function;
+        encodeURIComponent?: (str: string) => string;
```

Comment:
Maybe use `typeof encode` to show the relationship/default?

Reply:
Not going to create this binding, I think the property name already gives that away 😄